storybook/pages/DAppsWorkflowPage.qml

```diff
@@ -143,6 +143,15 @@ Item {
 
                     Layout.fillWidth: true
                 }
+
```

Comment:
I'd add a new page only for the WCUriInput.qml. The WorkFlow page is too big to accommodate all individual sub component states.